server/room.ts

```diff
@@ -158,6 +158,9 @@ export class Room {
       socket.on('CMD:playlistDelete', (data) =>
         this.playlistDelete(socket, data)
       );
+      socket.on('CMD:verifyAndEmitSubscriberStatus', (data) =>
```

Comment:
hmm, do we need a separate handler for this? We can just do it on each user connect right?

Reply:
or on CMD:uid where the user registers their UID